src/sentry/api/urls.py

```diff
@@ -1907,6 +1908,11 @@
                     SystemOptionsEndpoint.as_view(),
                     name="sentry-api-0-system-options",
                 ),
+                url(
+                    r"^beacon/$",
```

Comment:
Does this have an `internal` prefix automatically? If not, might be good to have that.

Reply:
It does have `internal` prefix